wallet/api/openrpc.json

```diff
@@ -2622,35 +2620,54 @@
         {
           "name": "wallet",
           "required": true,
+          "description": "The wallet from which the signing key comes from.",
           "schema": {
             "type": "string"
           }
         },
         {
           "name": "publicKey",
           "required": true,
+          "description": "The public key associated to the private key that is used to sign the transaction.",
           "schema": {
             "type": "string"
           }
         },
         {
           "name": "network",
+          "description": "The network configuration to use to retrieve the last block data. If specified, the last block data will be fetch by one of the nodes specified in this network configuration. Not required if the last block data is specified, as no network query will be issued.",
           "schema": {
             "type": "string"
           }
         },
         {
           "name": "transaction",
+          "description": "The transaction to sign.",
           "required": true,
           "schema": {
             "$ref": "#/components/schemas/transaction"
           }
         },
         {
           "name": "lastBlockData",
+          "description": "Instead of fetching the last block data from a network, it possible to manually specify it. If specified the parameters `network`, `retries` and `maximumRequestDuration` does not have to be specified.",
```

Comment:
```suggestion
          "description": "Instead of fetching the last block data from a network, it possible to manually specify it. If specified, then the parameters `network`, `retries` and `maximumRequestDuration` do not have to be specified.",
```